com.unity.netcode.gameobjects/Tests/Editor/Messaging/MessageReceivingTests.cs

```diff
@@ -0,0 +1,202 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+
+namespace Unity.Netcode.EditorTests
```

Comment:
oh my lord, I just want to trust you on tests. I'm sure they're alright and covering lots of things :)

Reply:
Most of the test changes are just updates to support the new APIs. Only a few of them are new. So I think it's fair to gloss over most of it.